crates/collab/migrations_llm.sqlite/20240806182921_test_schema.sql

```diff
@@ -0,0 +1,16 @@
+create table providers (
```

Comment:
As much as it pains me, I'm keeping the table names pluralized to match the tables in the existing collab database 😭

Would much prefer that they were singular.